DNSAuth/customers.go

```diff
@@ -2,46 +2,77 @@ package main
 
 import (
 	"database/sql"
-	"log"
-	"github.com/asergeyev/nradix"
+	"sync"
+	_ "github.com/go-sql-driver/mysql"
+	radix "github.com/armon/go-radix"
 	_"github.com/lib/pq"
 )
 
-type Customer struct {
-	Prefix string
-	Name string
-	PrefixMonit bool
-	ASNMonit bool
-}
+var DB_URL = "root:pass@(127.0.0.1)/customers"
```

Comment:
Again, this is a default value however since the `customer-db` field is required, this default value will never be used. Yes, I replaced postgres to use mysql instead but the URL for the database is still required. So I used the same `customer-db` field to declare the database URL. Finally, I'm not sure why you're mentioning removing the `influx-db` field, which is independent of postgres/mysql.

Reply:
All good on default value.  And I meant `customer-db`, not `influx-db`, sorry! Per comment above, we should `dnsauth.toml` so that it's not pointing to postgress:

```
customer-db = "postgres://user:password@127.0.0.1/postgres?sslmode=disable"
```